apis/config/v1beta1/imageconfig_types.go

```diff
@@ -55,6 +57,19 @@ type Images struct {
 	ConfigBuilder string `json:"config-builder"`
 }
 
+type DefaultImages struct {
```

Comment:
Do we want to also have default image components for system-logger and config-builder?

Should a tag be specified as part of the `ImageComponent.Repository` field?

Reply:
system-logger / config-builder are only available as ubi versions. Tag is not defined in the ImageComponent.Repository, this is only a struct to build using the ServerVersion.

If one wishes to override everything, then ServerImage (and eq) will not use any of these parameters.